core/templates/dev/head/header_js_libs.html

```diff
@@ -2,27 +2,10 @@
   <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
 <![endif]-->
 <!--[if gte IE 9]><!-->
-  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js">
+  <script src="/third_party/static/jquery-2.1.1/jquery.min.js">
```

Comment:
Aha -- is the idea that only jquery and angular need to be loaded at the top, and everything else can happen after that?